execution_engine/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to this project will be documented in this file.  The format
 
 ### Changed
 * Support building and testing using stable Rust.
+* Changed price of `create_purse` to 2.5CSPR to discourage people from creating purses in the payment code.
```

Comment:
The "in the payment code" is confusing b/c it increases the price across the whole execution, including session code. Is that correct @mpapierski ?

Reply:
Correct, but it was increased specifically to disallow creating those in the payment code.

Reply:
But the comment makes it sound as if the cost is increased only in the payment code. I don't think the last part needs to be included in the changelog.